warehouse/v0/util.js

```diff
@@ -29,7 +29,23 @@ function toSafeDBString(provider, name = "") {
   }
 }
 
-function safeTableName(provider, name = "") {
+function transformNameToBlendoCase(provider, name = "") {
```

Comment:
lets not make changes in v0 files. I don't think anybody is using this anymore

Reply:
removed v0/util